hw2/05-form.js

```diff
@@ -1 +1,34 @@
 // Add your code here
+const fullName = document.getElementById("fullName");
+const email = document.getElementById("email");
+const registrationStatus = document.getElementById("registrationStatus");
+const programmingLanguages = document.getElementById(
+  "programmingLanguages"
+).value;
+const operatingSystems = document.getElementById("operatingSystems").value;
+const fullStackWebDevelopment = document.getElementById(
+  "fullStackWebDevelopment"
+).value;
+const anythingElse = document.getElementById("anythingElse");
+
+const printInput = function printInput() {
+  console.log(`Full Name : ${fullName.value}`);
+  console.log(`Email : ${email.value}`);
+  console.log(`Registration Status : ${registrationStatus.value}`);
+  console.log("Courses taken :");
+  if (programmingLanguages.checked) {
+    console.log("Programming Languages");
+  }
+  if (operatingSystems.checked) {
+    console.log("Operating Systems");
+  }
+  if (fullStackWebDevelopment.checked) {
+    console.log("full Stack Web Development");
+  }
+  console.log(`Anything else? ${anythingElse.value}`);
+};
```

Comment:
Testing this on chrome and I can't seem to get it to output what classes I checked.

 I think the issue is because you grab the checkbox element's `value` at the top of the file, but then are trying to call `checked` on a value instead of an html element. Using the chrome debugger, I'm getting all these `<className>.checked` lines marked as `undefined`

Perhaps you could check for either the `on` or `null` values for checking the state of checkboxes? I'm sure there's other ways to do this as well.

Reply:
fixed.